src/store/index.ts

```diff
@@ -16,4 +18,12 @@ export type DesktopSettings = {
    * in the yaml config.
    */
   installState?: 'started' | 'installed' | 'upgraded';
+  devCpuMode?: boolean;
```

Comment:
Do we really need this flag? Can we compute this from `selectedDevice`? Or is there a difference between user cpu mode and dev cpu mode?

Reply:
If we need to pass extra arg to ComfyUI main.py, we can always set `launchArgs` in the user settings json.